packages/account/src/Containers/reset-trading-password.tsx

```diff
@@ -15,16 +15,15 @@ const ResetTradingPassword = observer(() => {
     } = ui;
     const location = useLocation();
     const query_params = new URLSearchParams(location.search);
-    const cfd_platform = /^trading_platform_(.*)_password_reset$/.exec(query_params.get('action') || '')?.[1];
-    const [platform] = React.useState(cfd_platform);
+    const platform = /^trading_platform_(.*)_password_reset$/.exec(query_params.get('action') || '')?.[1];
     const verification_code =
         platform === CFD_PLATFORMS.MT5
             ? client.verification_code.trading_platform_mt5_password_reset
             : client.verification_code.trading_platform_dxtrade_password_reset;
 
     return (
         <ResetTradingPasswordModal
-            platform={platform}
+            platform={platform as TPlatform}
```

Comment:
you change types of platform for ResetTradingPasswordModal, do we need here assertion anyway?

Reply:
yes. because the type of the variable platform computes to `string`. But the expected type is a specific set of strings